frontstage/views/passwords/reset_password.py

```diff
@@ -17,19 +17,18 @@
 def get_reset_password(token, form_errors=None):
     form = ResetPasswordForm(request.form)
 
-    url = app.config['VERIFY_PASSWORD_TOKEN']
-    parameters = {"token": token}
-    response = api_call('GET', url, parameters=parameters)
-
-    if response.status_code == 409:
-        logger.warning('Token expired', token=token)
-        return render_template('passwords/password-expired.html')
-    elif response.status_code == 404:
-        logger.warning('Invalid token sent to party service', token=token)
-        return redirect(url_for('error_bp.not_found_error_page'))
-    elif response.status_code != 200:
-        logger.error('Party service failed to verify token')
-        raise ApiError(response)
+    try:
+        party_controller.verify_token(token)
+    except ApiError as exc:
```

Comment:
Are exceptions caught by `try/except` statements before the app's error handlers?